ee/modules/030-cloud-provider-zvirt/images/cloud-controller-manager/src/pkg/cloudprovider/zvirt/instances.go

```diff
@@ -125,6 +125,10 @@ func (zc *Cloud) extractNodeAddressesFromVM(ctx context.Context, vm ovirtclient.
 		})
 	}
 
+	if len(localIP) < 1 {
+		localIP = externalIP
+	}
```

Comment:
```suggestion
    // If there are no local IP addresses, replace them with external ones
	if len(localIP) == 0 {
		localIP = externalIP
	}
```

Reply:
You should specify an explicit check condition to make it easier to read the code. And leave comments if you add non-standard behavior